tasks/utils.wdl

```diff
@@ -0,0 +1,50 @@
+version development
+
+'''
+additional useful tasks
+'''
+
+task GetGeneChrPairs {
```

Comment:
Could genericise this task name, because it's a generic csv pair extractor, for example:

```suggestion
task CsvPairExtractor {
```